.github/workflows/publish-helm.yml

```diff
@@ -0,0 +1,25 @@
+name: "Publish: helm chart"
```

Comment:
This workflow does not need to run on each main branch deployment, only on release (tags).

Reply:
ok